rest-api/tools/csv_importer.py

```diff
@@ -127,3 +136,23 @@ def _insert_entity(self, entity, existing_map, session, dry_run):
     if not dry_run:
       self.dao.insert_with_session(session, entity)
     return True
+
+  def _insert_new_participants(self, entity):
+    client = Client('rdr/v1', False, self.creds_file, self.instance)
```

Comment:
Before we start making client calls that create participants using the new sites, we're going to need to bounce the appengine instances, by calling "gcloud app instances list" followed by "gcloud app instances delete" for each instance (I think).

https://cloud.google.com/sdk/gcloud/reference/app/instances/list
https://cloud.google.com/sdk/gcloud/reference/app/instances/delete

Otherwise, the cache on the server won't necessarily contain the new sites.